server/app/services/migration/ProgramDefinitionUtils.java

```diff
@@ -0,0 +1,177 @@
+package services.migration;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.Optional;
+import services.program.BlockDefinition;
+import services.program.EligibilityDefinition;
+import services.program.ProgramDefinition;
+import services.program.ProgramQuestionDefinition;
+import services.program.predicate.AndNode;
+import services.program.predicate.LeafAddressServiceAreaExpressionNode;
+import services.program.predicate.LeafExpressionNode;
+import services.program.predicate.LeafOperationExpressionNode;
+import services.program.predicate.OrNode;
+import services.program.predicate.PredicateDefinition;
+import services.program.predicate.PredicateExpressionNode;
+import services.question.types.QuestionDefinition;
+
+/**
+ * Utility class for updating the program definition during migration. All methods should be static.
+ */
+final class ProgramDefinitionUtils {
```

Comment:
question (non-blocking): why make a utils class as well?

Reply:
should `convertNumberToSuffix` go in here?

Reply:
i think there are some other methods in `ProgramMigrationService` that could be static as well (anything that doesn't use an injected repository?), but i want to make sure i understand why we want to separate them out before making recommendations about what should go in here.

Reply:
Sounds good! Thank you for explaining.